### ISSUE TITLE
[Feature] Add Typewriter Text Animation-3

### ISSUE THREAD

**Issue by jaydeep099** [Rakesh9100/Beautiify#875]
### Description

The Text will be animated having Typewriter effect but different than the existing one. For reference you can check screenshot.

### Screenshots

![image](https://github.com/Rakesh9100/Beautiify/assets/148114163/49d05e7d-d270-411e-a1c9-31d64d7648c6)

### Checklist

- [X] I have checked the existing issues.
- [X] I follow [Contributing Guidelines](https://github.com/Rakesh9100/Beautiify/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.
- [X] I am a GSSOC'24 contributor.
- [ ] I am a IWOC'24 contributor.
- [ ] I am a JWOC'24 contributor.
- [ ] I am a SWOC'24 contributor.

### Code of Conduct

- [X] I agree to follow this project's "[Code of Conduct](https://github.com/Rakesh9100/Beautiify/blob/main/.github/CODE_OF_CONDUCT.md)".

**Comment by Aashka19** [comment]
please assign this issue to me under gssoc 24

**Comment by jaydeep099** [comment]
@Aashka19 I have been waiting for days to assign it to me. Please find another issue.

**Comment by Rakesh9100** [comment]
Go ahead @jaydeep099

**Comment by jaydeep099** [comment]
@Rakesh9100 Please give this issue a label

**Comment by jaydeep099** [comment]
@Rakesh9100 Please do it asap so i can make a PR

**Comment by Rakesh9100** [comment]
Pls provide your PR @jaydeep099 

Label will be added once the PR is merged!!

**Comment by jaydeep099** [comment]
@Rakesh9100 PR has been made please review it.
